server/src/main/java/com/warehousepro/exception/ApiException.java

```diff
@@ -0,0 +1,7 @@
+package com.warehousepro.exception;
+
+public class ApiException extends RuntimeException {
```

Comment:
Why you create `ApiException` class, I don't think other exceptions will get any benefit from this